### PR TITLE
Update usage messages from spotter

### PR DESCRIPTION
**Pull request by PaulWessel**
As per #5341.  Also fixed many inconsistencies in RST docs.

### CODE REVIEW

File: src/spotter/backtracker.c [hunk 1]
```diff
@@ -159,44 +159,57 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct BACKTRACKER_CTRL *C) {	/* De
 static int usage (struct GMTAPI_CTRL *API, int level) {
 	const char *name = gmt_show_name_and_purpose (API, THIS_MODULE_LIB, THIS_MODULE_CLASSIC_NAME, THIS_MODULE_PURPOSE);
 	if (level == GMT_MODULE_PURPOSE) return (GMT_NOERROR);
-	GMT_Message (API, GMT_TIME_NONE, "usage: %s [<table>] %s [-A[<young></old>]] [-Df|b]\n", name, SPOTTER_E_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[-F<driftfile] [-Lf|b<d_km>] [-M[<factor>]] [-N<upper_age>] [-Q<t_fix>] [-S<stem>] [-T<t_zero>]\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t[%s] [-W] [%s] [%s] [%s]\n\t[%s] [%s]\n\t[%s] [%s] [%s] [%s] [%s]\n\n",
-		GMT_V_OPT, GMT_b_OPT, GMT_d_OPT, GMT_e_OPT, GMT_h_OPT, GMT_i_OPT, GMT_o_OPT, GMT_q_OPT, GMT_s_OPT, GMT_colon_OPT, GMT_PAR_OPT);
+	GMT_Usage (API, 0, "usage: %s [<table>] %s [-A[<young>/<old>]] [-Df|b] [-F<driftfile>] [-Lf|b<d_km>] "
+		"[-M[<factor>]] [-N<upper_age>] [-Q<t_fix>] [-S<stem>] [-T<t_zero>] [%s] [-W[a|t]] [%s] [%s] [%s] "
```
Comment (meghanrjones) [inline src/spotter/backtracker.c:163]:
```suggestion
		"[-M[<factor>]] [-N<upper_age>] [-Q<t_fix>] [-S<stem>] [-T<t_zero>] [%s] [-W[a|t]] [%s] [%s] [%s] [%s] "
```

File: src/spotter/backtracker.c [hunk 2]
```diff
@@ -159,44 +159,57 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct BACKTRACKER_CTRL *C) {	/* De
...
+		"[%s] [%s] [%s] [%s] [%s] [%s] [%s]\n",
+		name, SPOTTER_E_OPT, GMT_V_OPT, GMT_b_OPT, GMT_d_OPT, GMT_e_OPT, GMT_h_OPT, GMT_i_OPT, GMT_o_OPT,
```
Comment (meghanrjones) [inline src/spotter/backtracker.c:165]:
```suggestion
		name, SPOTTER_E_OPT, GMT_V_OPT, GMT_b_OPT, GMT_d_OPT, GMT_e_OPT, GMT_f_OPT, GMT_h_OPT, GMT_i_OPT, GMT_o_OPT,
```

File: src/spotter/gmtpmodeler.c [hunk 3]
```diff
@@ -93,31 +93,36 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct GMTPMODELER_CTRL *C) {	/* De
...
+	GMT_Usage (API, 3, "s: Plate motion stage ID (1 is youngest).");
+	GMT_Usage (API, 3, "w: Rotation rate in degrees/Myr.");
+	GMT_Usage (API, 3, "x: Change in longitude since formation.");
+	GMT_Usage (API, 3, "y: Change in latitude since formation.");
+	GMT_Usage (API, 3, "X: Longitude at origin of crust.");
+	GMT_Usage (API, 3, "Y: Latitude at origin of crust.");
+	GMT_Usage (API, -2, "Default writes lon,lat,age,<adrswxyXY> to standard output.");
+	GMT_Usage (API, 1, "\n-T<time>");
+	GMT_Usage (API, -2, "Set fixed time of reconstruction to override any input ages.");
 	GMT_Option (API, "bi3,bo,d,e,h,i,o,q,s,:,.");
```
Comment (meghanrjones) [inline src/spotter/gmtpmodeler.c:126]:
```suggestion
	GMT_Option (API, "bi3,bo,d,e,f,h,i,o,q,s,:,.");
```

File: src/spotter/grdpmodeler.c [hunk 4]
```diff
@@ -100,38 +100,43 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct GRDROTATER_CTRL *C) {	/* Dea
 static int usage (struct GMTAPI_CTRL *API, int level) {
 	const char *name = gmt_show_name_and_purpose (API, THIS_MODULE_LIB, THIS_MODULE_CLASSIC_NAME, THIS_MODULE_PURPOSE);
 	if (level == GMT_MODULE_PURPOSE) return (GMT_NOERROR);
-	GMT_Message (API, GMT_TIME_NONE, "usage: %s <agegrdfile> %s [-F<polygontable>] [-G<outgrid>]\n", name, SPOTTER_E_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[%s] [%s] [-N<upper_age>] [-SadrswxyXY]\n\t[-T<time>] [%s] [%s] [%s]\n\t[%s] [%s] [%s]\n\n",
-		GMT_Id_OPT, GMT_Rgeo_OPT, GMT_V_OPT, GMT_b_OPT, GMT_d_OPT, GMT_h_OPT, GMT_r_OPT, GMT_PAR_OPT);
+	GMT_Usage (API, 0, "usage: %s <ingrid> %s [-F<polygontable>] [-G<outgrid>] [%s] [%s] [-N<upper_age>] "
+		"[-SadrswxyXY] [-T<time>] [%s] [%s] [%s] [%s] [%s] [%s]\n",
+		name, SPOTTER_E_OPT, GMT_Id_OPT, GMT_Rgeo_OPT, GMT_V_OPT, GMT_b_OPT, GMT_d_OPT, GMT_h_OPT, GMT_r_OPT, GMT_PAR_OPT);
```
Comment (meghanrjones) [inline src/spotter/grdpmodeler.c:105]: GMT_Id_OPT isn't included in the full usage message or online.

File: src/spotter/grdspotter.c [hunk 5]
```diff
@@ -210,36 +210,45 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct GRDSPOTTER_CTRL *C) {	/* Dea
 static int usage (struct GMTAPI_CTRL *API, int level) {
 	const char *name = gmt_show_name_and_purpose (API, THIS_MODULE_LIB, THIS_MODULE_CLASSIC_NAME, THIS_MODULE_PURPOSE);
 	if (level == GMT_MODULE_PURPOSE) return (GMT_NOERROR);
-	GMT_Message (API, GMT_TIME_NONE, "usage: %s <ingrid> -E[+]<rottable> -G<CVAgrid> %s\n", name, GMT_I_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t%s [-A<agegrid>] [-D[i|p]<grdfile>] [-L<IDgrid>] [-M]\n", GMT_Rgeo_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[-N<upper_age>] [-Q<IDinfo>] [-S] [-Tt|-u<age>] [%s] [-W<n_try]\n", GMT_V_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[-Z<z_min>[/<z_max>[/<z_inc>]]] [%s] [%s] [%s]\n\n", GMT_ho_OPT, GMT_r_OPT, GMT_PAR_OPT);
+	GMT_Usage (API, 0, "usage: %s <ingrid> %s -G<CVAgrid> %s %s [-A<agegrid>] [-D[i|p]<grdfile>] "
```
Comment (meghanrjones) [inline src/spotter/grdspotter.c:213]: -P is missing

File: src/spotter/polespotter.c [hunk 6]
```diff
@@ -94,29 +94,37 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct POLESPOTTER_CTRL *C) {	/* De
...
-		GMT_bi_OPT, GMT_d_OPT, GMT_e_OPT, GMT_h_OPT, GMT_i_OPT, GMT_r_OPT, GMT_o_OPT, GMT_s_OPT, GMT_colon_OPT, GMT_PAR_OPT);
+	GMT_Usage (API, 0, "usage: %s [%s] [-G<polegrid>] [%s] [-A<abyssalhills>] [-D<step>] [-Ea|f<sigma>] "
+		"[-F<FZfile>] [-N] [%s] [-Ss|p|l[<modifiers>]] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s]\n",
+		name, GMT_Id_OPT, GMT_Rgeo_OPT, GMT_V_OPT, GMT_bi_OPT, GMT_d_OPT, GMT_e_OPT, GMT_h_OPT,
+		GMT_i_OPT, GMT_r_OPT, GMT_o_OPT, GMT_s_OPT, GMT_colon_OPT, GMT_PAR_OPT);
 
 	if (level == GMT_SYNOPSIS) return (GMT_MODULE_SYNOPSIS);
 
+	GMT_Message (API, GMT_TIME_NONE, "  REQUIRED ARGUMENTS:\n");
+	GMT_Usage (API, 1, "\n%s", GMT_Id_OPT);
```
Comment (meghanrjones) [inline src/spotter/polespotter.c:105]: Is -I required? Synopsis and html suggest it is optional.

File: src/spotter/rotconverter.c [hunk 7]
```diff
@@ -125,30 +125,35 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct ROTCONVERTER_CTRL *C) {	/* D
 static int usage (struct GMTAPI_CTRL *API, int level) {
 	const char *name = gmt_show_name_and_purpose (API, THIS_MODULE_LIB, THIS_MODULE_CLASSIC_NAME, THIS_MODULE_PURPOSE);
 	if (level == GMT_MODULE_PURPOSE) return (GMT_NOERROR);
-	GMT_Message (API, GMT_TIME_NONE, "usage: %s [+][-] <rotA> [[+][-] <rotB>] [[+][-] <rotC>] ... [-A] [-D]\n", name);
-	GMT_Message (API, GMT_TIME_NONE, "\t[-F<out>] [-G] [-M[<factor>]] [-N] [-S] [-T] [%s] [-W]\n\t[%s] [%s]\n\n", GMT_V_OPT, GMT_h_OPT, GMT_PAR_OPT);
+	GMT_Usage (API, 0, "usage: %s [+][-] <rotA> [[+][-] <rotB>] [[+][-] <rotC>] ... [-A] [-D] n"
```
Comment (meghanrjones) [inline src/spotter/rotconverter.c:128]:
```suggestion
	GMT_Usage (API, 0, "usage: %s [+][-] <rotA> [[+][-] <rotB>] [[+][-] <rotC>] ... [-A] [-D] "
```

**Review by meghanrjones — APPROVED** [review]

File: src/spotter/grdspotter.c (hunk 5 above)
Comment (PaulWessel) [reply src/spotter/grdspotter.c:213]: There is no longer a -P but a -Di and -Dp.  I have updated the docs.

File: src/spotter/polespotter.c (hunk 6 above)
Comment (PaulWessel) [reply src/spotter/polespotter.c:105]: Thanks, -I is not required and now listed a such in usage.

File: src/spotter/grdpmodeler.c (hunk 4 above)
Comment (PaulWessel) [reply src/spotter/grdpmodeler.c:105]: Fixed.  Online also did not have -R.

**Pull request merged**